src/cpp/core/include/core/r_util/RSourceIndex.hpp

```diff
@@ -139,31 +141,33 @@ class RSourceItem
 
    // support for RSourceIndex::search
 
-   bool nameStartsWith(const std::string& term, bool caseSensitive) const
+   bool nameStartsWith(const std::string& term, bool caseSensitive, bool includeTestItems) const
```

Comment:
I don't think this is the right place to add the `bool includeTestItems` parameter; shouldn't that decision be made before `nameStartsWith()` is called?